app/components/vis-bug/vis-bug.element.js

```diff
@@ -230,6 +231,18 @@ export default class VisBug extends HTMLElement {
     }
   }
 
+  execCommand(command) {
+    const query = `/${command}`
+
+    if (PluginRegistry.has(query))
+      return PluginRegistry.get(query)({
+        selected: this.selectorEngine.selection(),
+        query
+      })
+
+    return Promise.resolve(new Error("Query not found"))
```

Comment:
this doesnt look like it's returning `{}`, i wonder why a bad exec doesnt show the error?

Reply:
I expect it's because of the way ava + puppeteer handle promises in evaluate? I tried search for some docs on that but couldn't find them (quickly) so just went with this. The `{}` is consistent for invalid queries so :man_shrugging:.